core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.ts

```diff
@@ -92,6 +92,11 @@ export class TopicEditorTabComponent implements OnInit, OnDestroy {
   uncategorizedEditOptionsIndex: number;
   subtopicEditOptionsAreShown: number;
   skillOptionDialogueBox: boolean = true;
+  MAX_CHARS_IN_TOPIC_NAME: number;
+  MAX_CHARS_IN_TOPIC_DESCRIPTION: number;
+  maxWebTitleFrag: number;
+  minWebTitleFrag: number;
```

Comment:
Where are these being assigned values?

Reply:
These were previously directly assigned in **topic-editor-tab.directive.html** . But these values must be defined in JS file also, so I added them in **topic-editor-tab.directive.html.ts** file.